projects/kubernetes/kops/Makefile

```diff
@@ -0,0 +1,53 @@
+BASE_DIRECTORY:=$(shell git rev-parse --show-toplevel)
+PROJECT_DIRECTORY:=$(BASE_DIRECTORY)/projects/kubernetes/kops
+CLONED_REPO_DIRECTORY:=$(PROJECT_DIRECTORY)/kops
```

Comment:
is this supposed to be `$(BASE_DIRECTORY)/projects/kubernetes/kops/kops`?

Reply:
yeah, it's the dir of the cloned kops repo, which we clone into that path